src/main/scala/io/sdkman/changelogs/DetektMigrations.scala

```diff
@@ -0,0 +1,26 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "079")
```

Comment:
I think it should be 080. I just noticed it

Reply:
That's probably bc the PR was open for too long. Can we still change it?

Reply:
I wouldn't bother too much because the migration succeeded.